sdk/onlineexperimentation/analytics-onlineexperimentation-rest/EXAMPLES.md

```diff
@@ -0,0 +1,561 @@
+# Online Experimentation Rest Client - Examples
```

Comment:
is there any reason to not include some examples in README.md?  One benefit of having them in README is that customers can look at npmjs package landing page and see basic examples immediately.

Reply:
Good call, added a simple example for initializing and using the library. The full set of examples seemed too long to put inline in the main readme.